src/sconsx/tools/python.py

```diff
@@ -38,7 +38,10 @@ def default(self):
 
       self._default['include'] = get_python_inc(plat_specific=1)
       if CONDA_ENV:
-        self._default['libpath'] = pj(CONDA_PREFIX, 'lib')
+          if os.name == 'posix':
+              self._default['libpath'] = pj(CONDA_PREFIX, 'lib')
+          else:
+              self._default['libpath'] = pj(CONDA_PREFIX, 'libs')            

```

Comment:
I think that for CONDA, Python is also set in the environment variables

Reply:
The lib of python (python27.dll) is in the libs directory and is not set correctly by default by sconsx on windows.

Reply:
Thanks. These lines are also correct outside conda build.